aws/resource_aws_kinesis_firehose_delivery_stream.go

```diff
@@ -1192,6 +1194,7 @@ func resourceAwsKinesisFirehoseDeliveryStream() *schema.Resource {
 
 						"s3_backup_mode": {
 							Type:     schema.TypeString,
+							ForceNew: true,
```

Comment:
This change is okay. [`ElasticsearchDestinationUpdate`](https://docs.aws.amazon.com/firehose/latest/APIReference/API_ElasticsearchDestinationUpdate.html) does not support updating this parameter 👍